pywinauto/controls/uia_controls.py

```diff
@@ -229,8 +229,10 @@ def item_count(self):
 class EditWrapper(UIAWrapper.UIAWrapper):
 
     """Wrap an UIA-compatible Edit control"""
+    # TODO: this class supports only 1-line textboxes so there is no point
+    # TODO: in methods such as line_count(), line_length(), get_line(), etc
 
-    controltypes = [
+    control_types = [
```

Comment:
Hmm... Probably we need some warning in metaclass to detect such cases. Good catch, Alexander.